web/src/components/__tests__/ChatConversation.spec.tsx

```diff
@@ -37,7 +51,29 @@ describe('ChatConversation', () => {
     const { getByText, getByTestId } = render(testMessages, false)
     expect(getByText('chat:initialMessage')).toBeTruthy()
     expect(getByTestId(testMessages[0]!.id)).toBeTruthy()
+    expect(getByTestId(testMessages[2]!.id)).toBeTruthy()
+  })
+
+  it('should display typing indicator before the initial automatic answer and after for 60 seconds', () => {
+    const { getByText, queryByText, getByTestId } = render(testMessages, false)
+    expect(getByTestId(testMessages[0]!.id)).toBeTruthy()
+    expect(getByText('...')).toBeTruthy()
     expect(getByTestId(testMessages[1]!.id)).toBeTruthy()
+    expect(getByText('...')).toBeTruthy()
+
+    act(() => {
+      jest.advanceTimersByTime(TYPING_INDICATOR_TIMEOUT)
+    })
```

Comment:
Tip: You can just use this to avoid exporting/importing the actual length of the timeout (same below):
```suggestion
    act(() => jest.runAllTimers())
```